src/core/webidl.js

```diff
@@ -284,12 +283,13 @@ function renderWebIDL(idlElement) {
   try {
     parse = webidl2.parse(idlElement.textContent);
   } catch (e) {
-    pub(
-      "error",
-      `Failed to parse WebIDL: ${e.message}.
-      <details>
-      <pre>${idlElement.textContent}\n ${e}</pre>
-      </details>`
+    showInlineError(
+      idlElement,
+      `Failed to parse WebIDL: ${e.bareMessage}.`,
+      e.bareMessage,
+      {
+        details: `<pre>${e.context}</pre>`,
```

Comment:
Nit: Can we do this instead?
``` js
{ details: `<pre>${e.context}</pre>` }
```

Reply:
ESLint makes me do the longer way 🤔

Reply:
Oh, it seems somehow it allows both ways. I'll apply your suggestion.